package.json

```diff
@@ -61,7 +61,7 @@
     "eslint-plugin-n": "^15.2.5",
     "eslint-plugin-promise": "^6.0.1",
     "prettier": "2.7.1",
-    "puppeteer": "^19.2.0",
+    "puppeteer": "^19.11.1",
```

Comment:
just asking, is it required?

Reply:
nope, I thought one of my problem was tied to puppeteer version, but not. I kept this change because it's the latest ABI-compatible version though